jmeter-service/jmeterUtils.go

```diff
@@ -134,17 +156,29 @@ func parseJMeterResult(jmeterCommandResult string, testInfo *TestInfo, workload
  * Error: error details if status was false
  */
 func executeJMeter(testInfo *TestInfo, workload *Workload, resultsDir string, url *url.URL, LTN string, funcValidation bool) (bool, error) {
-	os.RemoveAll(resultsDir)
+	err := removeAllDirPath(resultsDir)
+	if err != nil {
+		return false, err
```

Comment:
Would it make sense to create new test cases where all these new `return` statements of the function are checked?

Reply:
I can create unit tests for the new functions `makeAllDirPath(...)` and `removeAllDirPath(...)`, other changes are just checks of return values for errors. Is this what you are pointing at?

Reply:
Note, that this part of jmeter service is currently experiencing some restructuring as part of #5916

Reply:
Ok, so do you prefer to wait with merging of this PR?

Reply:
Or at least we can remove the changes of jmeter service from this PR and look at it afterwards...

Reply:
yeah sounds good

Reply:
@warber do you prefer removing all changes of jmeter-service or just the ones in jmeterUtils.go? Thanks

Reply:
lets drop all in jmeter-service